README.md

```diff
@@ -1,152 +1,335 @@
 # Minio Quickstart Guide
 [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
 
-Minio is an object storage server released under Apache License v2.0. It is compatible with Amazon S3 cloud storage service. It is best suited for storing unstructured data such as photos, videos, log files, backups and container / VM images. Size of an object can range from a few KBs to a maximum of 5TB.
+Minio consists of an *object storage server*, released under Apache License v2.0, that is compatible with the Amazon S3 cloud storage service, as well as a *client component* used for sending and retrieving objects to and from the server. 
```

Comment:
@jamesneimanconsulting  we want to articulate here what can be done with Minio server. So i think we need the part that it is an object storage server and this is what you do with an object storage server in the beginning intro.

Reply:
@deekoder I have updated the branch with this change.